vaadin-testbench-core/src/test/java/com/vaadin/testbench/unit/NotificationWrapTest.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.testbench.unit;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import com.vaadin.flow.component.notification.Notification;
+
+class NotificationWrapTest extends UIUnitTest {
+
+    @Override
+    protected String scanPackage() {
+        return "com.example";
+    }
+
+    @Test
+    void notOpenedNotification_isNotUsable() {
+        Notification notification = new Notification("Not Opened");
+
+        NotificationWrap<?> notification_ = $(NotificationWrap.class,
+                notification);
+
+        Assertions.assertFalse(notification_.isUsable(),
+                "Not opened Notification shouldn't be usable");
+    }
+
+    @Test
+    void openedNotification_isUsable() {
+        String notificationText = "Opened notification";
+        Notification notification = Notification.show(notificationText);
+        ComponentWrap.flushChanges();
```

Comment:
Would be nice if we could somehow run this automatically when ever something is set to `beforeClientResponse`, but don't know how complex that would be.

Reply:
added note for this